samples/tfm_integration/psa_firmware/boards/hello-an547.hex

```diff
@@ -0,0 +1,3614 @@
+:020000040106F3
```

Comment:
How difficult would it be to Build and sign this during CI rather than checking in the hex file?

Reply:
Probably a case for `extra_post_build_commands` in the project cmake file? (Ex.: https://github.com/zephyrproject-rtos/zephyr/blob/main/modules/trusted-firmware-m/CMakeLists.txt#L458) ... but given that this file exists purely for CI, personally I'm OK with having the pre-built image unless you feel strongly we should automate this.

Reply:
It has to be built _before_ your build. I'll punt on this for now. If you would like, opening an issue to improve this would be fine with me.